src/video/surface_batch.hpp

```diff
@@ -39,8 +39,11 @@ class SurfaceBatch
   std::vector<Rectf> move_dstrects() { return std::move(m_dstrects); }
   std::vector<float> move_angles() { return std::move(m_angles); }
 
+  Color get_color() { return m_color; }
```

Comment:
const

Reply:
I didn't even remember I edited that file